src/configuration/xcWebSocketBridgeConfiguration.ts

```diff
@@ -7,6 +7,7 @@ export const Kinds = {
 
 
 export enum Commands {
+    hb,
```

Comment:
Server receives "hb" string rather than "heartbeat" string for performance issues. So hb enum corresponds to "hb" string.